### PR TITLE
docs: Updating install documentation related to Docker memory allocation

### PR DESCRIPTION
**Pull request by markgrover**
### Summary of Changes

Added an image for docker memory allocation update. Also, added a suggestion when starting docker for first-timers to proactively go through troubleshooting steps to save pain later.

### CheckList

Make sure you have checked **all** steps below to ensure a timely review.

- [x ] PR title addresses the issue accurately and concisely, including a title prefix.
- [x] PR includes a summary of changes.
- [x] My commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ The following instructions are for setting up a version of Amundsen using Docker
    ```bash
    $ git clone --recursive git@github.com:amundsen-io/amundsen.git
    ```
-3. Enter the cloned directory and run:
+3. Enter the cloned directory and run below. If it's your first time, you may want to proactively go through [troubleshooting](troubleshooting) steps, especially the first one related to heap memory for ElasticSearch and Docker engine memory allocation (leading to Docker error 137):
```
Comment (dorianj) [inline docs/installation.md:11]: Suggest putting this immediately below the code block

**Review by dorianj — APPROVED** [review]
@markgrover DCO, one minor comment

**Comment by dorianj** [comment]
@markgrover please rebase to fix git history

File: docs/installation.md (hunk 1 above)
Comment (markgrover) [reply docs/installation.md]: Thanks updated!

**Pull request merged**